css-brochure-form/styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+}
+
+form {
+  width: 50%;
+  margin: 0 auto;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+  justify-items: center;
```

Comment:
These can be removed, this is not a flexbox, so these properties aren't doing anything